docs/quick_start.rst

```diff
@@ -21,3 +24,4 @@ Output:
  * output.nwb: NWB file including spike times
```

Comment:
minor: I have noticed that the latest code instead creates input_json and output_json. @NileGraddis. Could we may be make them input.json and output.json instead (i.e., to have an extension .json)? In any case, we need the documentation to match the JSON file names.